README.md

```diff
@@ -93,18 +95,11 @@ $ aws cloudformation create-stack \
         ParameterKey=SNSSubscriptionEmail,ParameterValue="email-address" \
         ParameterKey=IntegrationHttpEndpoint,ParameterValue='"endpoint-url"'
 ```
-
-## 3. Configure SSL/HTTPS  
```

Comment:
Renee and I were thinking we should not modify the existing ReadMe and CFN template which is there for customers to try out with their own target service integration.

However this could still be a good addition to existing templates and ReadMe's. We already have a CFN template demonstrating a mock service integration using Lambda.